test/solveEquation/solveEquation.test.js

```diff
@@ -96,6 +96,8 @@ describe('solveEquation for =', function () {
     ['2/(1 + 1 + 4x) = 1/3', 'x = 1'],
     ['(3 + x) / (x^2 + 3) = 1', 'x = [0, 1]'],
     ['6/x + 8/(2x) = 10', 'x = 1'],
+    // Use of nested scope (i.e., baz depends on bar)
```

Comment:
oo also could you test substeps to show how things get sub'd in one at a time? there are some tests that already test substeps you can probably base off of but let me know if you want help setting that up